### ISSUE TITLE
Warn about `sudo cd`,`sudo export` and similar

### ISSUE THREAD

**Issue by koalaman** [koalaman/shellcheck#1137]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
sudo cd /root

```

#### Here's what shellcheck currently says:

Nothing

#### Here's what I wanted or expected to see:

Can't use `sudo` with `cd`
